deliverables/ObligatoriskOppgave2.md

```diff
@@ -0,0 +1,157 @@
+Criteria: Once the game has initialized and we have created a set amount of players, attempting to move a robot of the grid itself should not work. Neither should players be able to move onto tiles already occupied unless they have cards that can for example push other robots away.
+## Developer Stories 
+
+Because of how little the user will see of the work this sprint we also did a few acceptance criteria specifically for us developers.
+* **Criteria**: The game should abstract away locations and the underlying grid and use that to represent elements on the board, not the other way around.
+  * We need a way to keep track of the position of elements, and a grid that matches the entire board.
+* **Criteria**: The game should have a clear hierarchy of inheritance for elements, with a single root interface that is inherited down through the specific elements placed on the grid.
+  * A robot is an element, and so is a laser or a flag. As such they should inherit a common interface exposing the minimally required methods for an element.
+* **Criteria**: A player should control one and only one robot.
+  * A player has HP and a robot takes damage. We need to ensure that a player can never control more than one robot at a time and that damage inflicted (and subsequently the death of a robot) affects his or hers HP.
+
+
+## Testing 
+
+We have made several test classes with several test methods, to ensure that what we have developed is functioning properly. `DirectionTest` checks if every direction to the left, right and the opposite is correct. `GridTest` checks if the proper coordinates are returned
+
+One for directions, the grid, position, player, robotest
+
+
+
+For hvert krav dere jobber med, må dere lage 1) ordentlige brukerhistorier, 2) akseptansekriterier og 3) arbeidsoppgaver. Husk at akseptansekriterier ofte skrives mer eller mindre som tester ✔️
+
+Dersom dere har oppgaver som dere skal til å starte med, hvor dere har oversikt over både brukerhistorie, akseptansekriterier og arbeidsoppgaver, kan dere ta med disse i innleveringen også. ✔️
+
+Forklar kort hvordan dere har prioritert oppgavene fremover 
+* In order to ensure we complete the assignment in a timely and effective manner, we prioritized the tasks as follows;
+  * At this moment, the backend and the frontend parts of the game live in different worlds. As we have finished a lot of the backend that we consider necessary, the goal now will be to connect the map with the backend.
+    * Get the map connected to the current back end
+    * Create a proper GUI for the game.
+    * Program cards
+    * We will be prioritizing this before working on the logic for the actual game.
+
+Forklar kort hvilke hovedkrav dere anser som en del av MVP og hvorfor. Hvis det er gjort endringer i rekkefølge utfra hva som er gitt fra kunde, hvorfor er dette gjort?
+* We believe the following hovedkrav are a part of MVP;
+  * Krav 
+    * Reasoning
```

Comment:
We can keep the priority list of requirements from the first assignment, but make them a little more specific.
* Show a game board that is connected to the current backend of the game
* Show elements like the player and flag that are connected to the current backend game